vizro-core/src/vizro/static/css/bootstrap_overwrites.css

```diff
@@ -4,3 +4,7 @@ but do not want to take over to `vizro-bootstrap` as these settings might not be
 .card .nav-link {
   height: 100%;
 }
+
+.form-check:last-of-type {
+  margin-bottom: 0;
+}
```

Comment:
@pruthvip15 - should this live here or in the bootstrap theme? Both are fine for me, even though I would prefer the bootstrap theme. I am just not sure if it's generic enough to live there?